### PR TITLE
Correct AMM Details

### LINKED ISSUE #2497: `TradingFee` is missing from `AMMDeposit`
Source code proof: https://github.com/XRPLF/rippled/blob/develop/src/ripple/protocol/impl/TxFormats.cpp#L103

### LINKED ISSUE #2569: AMM content described in "Future Developments" should be updated
https://xrpl.org/docs/use-cases/defi/algorithmic-trading/#future-developments
スクリーンショット 2024-04-28 22 55 07

### PR DESCRIPTION
**Pull request by oeggert**
closes #2497
closes #2569

### CODE REVIEW

File: docs/use-cases/defi/algorithmic-trading.md [hunk 1]
```diff
@@ -84,12 +84,12 @@ If your approach involves following other transaction activity, you may need to
...
 
-### Future Developments
+### Automated Market Makers
 
-Ripple has proposed extending the XRP Ledger protocol with a native Automated Market Maker (AMM) design ([XLS-30](https://github.com/XRPLF/XRPL-Standards/blob/master/XLS-0030-automated-market-maker/README.md)) that would work alongside the existing central limit order based (CLOB) decentralized exchange. If this proposal is accepted and becomes enabled as an [amendment](../../concepts/networks-and-servers/amendments.md), AMMs will become an important factor in trading on the XRP Ledger. You can read more at the following links:
+Ripple natively supports Automated Market Makers (AMM) that work alongside the existing central limit order based (CLOB) decentralized exchange. AMMs are an important factor in trading on the XRP Ledger. You can read more at the following links:
```
Comment (intelliot) [inline docs/use-cases/defi/algorithmic-trading.md]:
```suggestion
The XRP Ledger natively supports Automated Market Makers (AMM) that work alongside the existing central limit order based (CLOB) decentralized exchange. AMMs are an important factor in trading on the XRP Ledger. You can read more at the following links:
```

File: docs/use-cases/defi/algorithmic-trading.md [hunk 2]
```diff
@@ -84,12 +84,12 @@ If your approach involves following other transaction activity, you may need to
...
 
-- [XLS-30d: Automated Market Maker standards proposal](https://github.com/XRPLF/XRPL-Standards/discussions/78)
-- [AMM documentation](../../concepts/tokens/decentralized-exchange/automated-market-makers.md)
+- [Automated Market Makers](/docs/concepts/tokens/decentralized-exchange/automated-market-makers)
+- [Use the AMM Auction Slot for Lower Fees](/docs/tutorials/javascript/trade-on-ledger/use-amm-auction-slot-for-lower-fees)
```
Comment (intelliot) [inline docs/use-cases/defi/algorithmic-trading.md:92]:
```suggestion
- [Automated Market Makers](/docs/concepts/tokens/decentralized-exchange/automated-market-makers)
- [Use the AMM Auction Slot for Lower Fees](/docs/tutorials/javascript/trade-on-ledger/use-amm-auction-slot-for-lower-fees)
- [XLS-30 Specification](https://github.com/XRPLF/XRPL-Standards/tree/master/XLS-0030-automated-market-maker#readme)
```
Personally I think the specification is still a useful reference, but it does not need to be the 1st bullet.

File: docs/use-cases/defi/algorithmic-trading.md [hunk 3]
```diff
@@ -84,12 +84,13 @@ If your approach involves following other transaction activity, you may need to
...
 
-### Future Developments
+### Automated Market Makers
 
-Ripple has proposed extending the XRP Ledger protocol with a native Automated Market Maker (AMM) design ([XLS-30](https://github.com/XRPLF/XRPL-Standards/blob/master/XLS-0030-automated-market-maker/README.md)) that would work alongside the existing central limit order based (CLOB) decentralized exchange. If this proposal is accepted and becomes enabled as an [amendment](../../concepts/networks-and-servers/amendments.md), AMMs will become an important factor in trading on the XRP Ledger. You can read more at the following links:
+The XRP Ledger natively supports Automated Market Makers (AMM) that work alongside the existing central limit order based (CLOB) decentralized exchange. AMMs are an important factor in trading on the XRP Ledger. You can read more at the following links:
```
Comment (DennisDawson) [inline docs/use-cases/defi/algorithmic-trading.md]: suggest Automated Market Makers (AMMs)

**Review by DennisDawson — APPROVED** [review]
Approved with a teensy suggestion.

**Pull request merged**